_layouts/docwithnav.html

```diff
@@ -173,12 +137,11 @@
 						    }
 						});
 					}
-					if(!isCode){//dont change/animate nav menu when dispalying a code sample
-						if(clickedon.parents().hasClass('bodywrapper')){//animate navigation menu
-							animatenav(findcurnavitem());//is an in-page link
-						} else {
-							animatenav(clickedon);//is a nav menu link
+					if(!isCode){//animate nav menu, but not when dispalying a code sample
+						if(clickedon.parents().hasClass('bodywrapper')){//if its an in-page link, first find the nav menu item
+							clickedon = findcurnavitem();
 						}
+						animatenav(clickedon, isAnchor);
```

Comment:
add the "isAnchor" attribute to stop anchors from toggling parent containers in the nav

i wont call them out but you'll also see this attribute added in two other places below